### ISSUE TITLE
Attributes: Filters

### ISSUE THREAD

**Issue by callmekatootie** [topcoder-platform/u-bahn-api#4]
### Question 1

@cwdcwd Which API to call to get the list of company attributes under the filter modal:

![image](https://user-images.githubusercontent.com/2317133/83967934-1c0edc80-a8e3-11ea-9cd3-770151c2cc0b.png)

I am guessing that we don't have 1 api for this. We need to create a new endpoint, that:
- will get the org of the caller
- get the attribute groups for the org
- get all attributes that belong to this attribute group

The UI will:
- Filter `location`, `title` and `isAvailable` attributes from this
- and display the remaining attributes under Company Attributes

Is that right? Or some other approach...

### Question 2

Once we have the attributes to display in the filter, when the user activates them, in the left sidebar, we will show that filter. Question is how does the user specify the value for this filter? As in, in the UI, how dos the user specify values for each of the attributes selected

**Comment by cwdcwd** [comment]
@callmekatootie yes, I think that's the right approach. We should be able to take the UID from the JWT, which will be their TC UID and be present in the `ExternalProfile` with the OrgId.

I defer on question to to Will on how that should be working

**Comment by callmekatootie** [comment]
@wdprice Just making sure you have seen this - would like to know the answer to Question 2 above.

**Comment by wdprice** [comment]
Type ahead search for the value, no tag cloud or pills needed.
